### PR TITLE
Update doc to prepare for 1proposal

### PR DESCRIPTION
**Pull request by LaraAS**

### CODE REVIEW

File: docs/developer-docs/integrations/sns/tokenomics/preparation.md [hunk 1]
```diff
@@ -62,6 +58,18 @@ ledger canister such as the token name, token symbol, and the ledger transaction
       swap tokens are reserved for future swaps (separate from the treasury). 
       Note that future swap are not yet designed, but having the tokens reserved
       makes SNSs forward compatible to such a feature if it is added in the future.
+
+<!-- NEW BULLET POINT:
+**Decentralization swap parameters**: this defines the parameters of the decentralization swap. MAKE SUB-BULLETS
+**Rules of the swap** such as the minimum number of ICP tokens that the swap must collect to be successful and the maximum number of ICP tokens that it will collect. 
+**Conditions of swap participation** such as the definition of a text that must be confirmed by users when participating in the swap and the possibility to define countries from which users cannot participate (by IP addresses).
+**Fallback controllers of the dapp:** Another important parameter is the set of fallback controllers of the dapp.
```
Comment (LaraAS) [inline docs/developer-docs/integrations/sns/tokenomics/preparation.md:66]: @DanielThurau could you please confirm that this is still true with 1proposal?

File: docs/developer-docs/integrations/sns/integrating/frontend-integration.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# SNS frontend integration
+## Overview
+There are different reasons why you might want to interface with an SNS project.
+For example, you might want to integrate some of the SNS DAO functionality in the frontend of you dapp canister such that the dapp users can also vote on relevant SNS proposals (that goveran and change the dapp) directly in the same application.
```
Comment (peterpeterparker) [inline docs/developer-docs/integrations/sns/integrating/frontend-integration.md:4]: > in the frontend of you dapp canister

Typo (you vs your)
I would also shorten and just say "your dapp", drop "canister"

> goveran

Typo

File: docs/developer-docs/integrations/sns/integrating/frontend-integration.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+However, you might also consdier interfacing with SNSs if you work on a project that builds a wallet dapp or DEX, for example to provide an interface to the SNS swap, the SNS governance, or the SNS ledger.
```
Comment (peterpeterparker) [inline docs/developer-docs/integrations/sns/integrating/frontend-integration.md:5]: > consdier

Typo

File: docs/developer-docs/integrations/sns/integrating/frontend-integration.md (hunk 2 above)
Comment (LaraAS) [reply docs/developer-docs/integrations/sns/integrating/frontend-integration.md:4]: Thanks. fixed locally and will be included in next commit.

File: docs/developer-docs/integrations/sns/integrating/frontend-integration.md (hunk 3 above)
Comment (LaraAS) [reply docs/developer-docs/integrations/sns/integrating/frontend-integration.md:5]: fixed

File: docs/developer-docs/integrations/sns/launching/launch-steps.md [hunk 4]
```diff
@@ -1,9 +1,15 @@
-# SNS launch steps
+# Commands & actions to go through SNS launch
+
+<!-- 
+CHANGE TITLE: Commands & actions to go through SNS launch - legacy flow 
+NEW: INFO 
+This page refers to a legacy flow of how an SNS is launched. This is still supported at the moment but might be deprecated going forward. It is recommended to use the new SNS launch flow which can be fund [here](./launch-steps-1proposal.md). Please find more context on this topic [here](./index.md).
```
Comment (jessiemongeon1) [inline docs/developer-docs/integrations/sns/launching/launch-steps.md:6]:
```suggestion
This page refers to a legacy flow of how an SNS is launched. This is still supported at the moment but might be deprecated going forward. It is recommended to use the new SNS launch flow which can be found [here](./launch-steps-1proposal.md). Please find more context on this topic [here](./index.md).
```

File: docs/developer-docs/integrations/sns/launching/launch-summary.md [hunk 5]
```diff
@@ -1,4 +1,11 @@
 # Stages of an SNS launch
+<!--
```
Comment (jessiemongeon1) [inline docs/developer-docs/integrations/sns/launching/launch-summary.md:2]:
```suggestion
<!---
```

File: docs/developer-docs/integrations/sns/launching/launch-steps.md [hunk 6]
```diff
@@ -1,9 +1,15 @@
-# SNS launch steps
+# Commands & actions to go through SNS launch
+
+<!--
```
Comment (jessiemongeon1) [inline docs/developer-docs/integrations/sns/launching/launch-steps.md:3]:
```suggestion
<!---
```

File: docs/developer-docs/integrations/sns/tokenomics/preparation.md [hunk 7]
```diff
@@ -73,17 +81,15 @@ The developer and airdrop neurons are the only neurons that exist when
...
-
+Even though the governance canister is in a special pre-decentralization mode, some proposals can already be used, for example to upgrade the dapp canister(s) or to register new kinds of proposals. 
+Therefore, **if you want to use SNS proposals at this stage, for example to ensure that the dapp can be upgraded 
+at any time, you must make sure that enough of the initial neurons can vote to reach a decison.**
+See [Stage 1 of the launch](../launching/launch-summary.md) for more details of this. 
 :::
 
-All developers, airdrop principals, and the swap participants receive
-their neurons as a *basket of neurons*.
+<!-- NEW: change link above -->
```
Comment (jessiemongeon1) [inline docs/developer-docs/integrations/sns/tokenomics/preparation.md:90]:
```suggestion
<!--- NEW: change link above --> 
```

File: docs/developer-docs/integrations/sns/tokenomics/preparation.md (hunk 1 above)
Comment (DanielThurau) [reply docs/developer-docs/integrations/sns/tokenomics/preparation.md:66]: I'm not entirely sure what I'm confirming among all of these bullet points but I read them all and nothing in them is false...so confirmed :)

File: docs/developer-docs/integrations/sns/launching/launch-steps.md [hunk 8]
```diff
@@ -1,9 +1,15 @@
-# SNS launch steps
+# Commands & actions to go through SNS launch
+
```
Comment (jessiemongeon1) [inline docs/developer-docs/integrations/sns/launching/launch-steps.md:2]:
```suggestion
```

File: docs/developer-docs/integrations/sns/launching/launch-steps.md [hunk 9]
```diff
@@ -1,9 +1,15 @@
...
+<!---
```
Comment (jessiemongeon1) [inline docs/developer-docs/integrations/sns/launching/launch-steps.md:3]:
```suggestion


<!---
```

File: docs/developer-docs/integrations/sns/launching/launch-summary.md [hunk 10]
```diff
@@ -1,4 +1,11 @@
 # Stages of an SNS launch
+<!---
```
Comment (jessiemongeon1) [inline docs/developer-docs/integrations/sns/launching/launch-summary.md:2]:
```suggestion


<!---
```

File: docs/developer-docs/integrations/sns/tokenomics/preparation.md [hunk 11]
```diff
@@ -73,17 +81,15 @@ The developer and airdrop neurons are the only neurons that exist when
...
-
+Even though the governance canister is in a special pre-decentralization mode, some proposals can already be used, for example to upgrade the dapp canister(s) or to register new kinds of proposals. 
+Therefore, **if you want to use SNS proposals at this stage, for example to ensure that the dapp can be upgraded 
+at any time, you must make sure that enough of the initial neurons can vote to reach a decison.**
+See [Stage 1 of the launch](../launching/launch-summary.md) for more details of this. 
 :::
 
-All developers, airdrop principals, and the swap participants receive
-their neurons as a *basket of neurons*.
+<!--- NEW: change link above -->
```
Comment (jessiemongeon1) [inline docs/developer-docs/integrations/sns/tokenomics/preparation.md:90]:
```suggestion


<!--- NEW: change link above --> 
```

File: docs/developer-docs/integrations/sns/launching/launch-steps.md [hunk 12]
```diff
@@ -1,9 +1,16 @@
-# SNS launch steps
+# Commands & actions to go through SNS launch
+
+
+<!---
```
Comment (jessiemongeon1) [inline docs/developer-docs/integrations/sns/launching/launch-steps.md:4]:
```suggestion

```

File: docs/developer-docs/integrations/sns/launching/launch-steps.md [hunk 13]
```diff
@@ -1,9 +1,16 @@
...
+CHANGE TITLE: Commands & actions to go through SNS launch - legacy flow
```
Comment (jessiemongeon1) [inline docs/developer-docs/integrations/sns/launching/launch-steps.md:5]:
```suggestion
[//]: # CHANGE TITLE: Commands & actions to go through SNS launch - legacy flow 
```

File: docs/developer-docs/integrations/sns/launching/launch-steps.md [hunk 14]
```diff
@@ -1,9 +1,16 @@
-# SNS launch steps
+# Commands & actions to go through SNS launch
+
+
+<!---
+CHANGE TITLE: Commands & actions to go through SNS launch - legacy flow 
+NEW: INFO
```
Comment (jessiemongeon1) [inline docs/developer-docs/integrations/sns/launching/launch-steps.md:6]:
```suggestion
[//]: # NEW: INFO 
```

File: docs/developer-docs/integrations/sns/launching/launch-steps.md [hunk 15]
```diff
@@ -1,9 +1,16 @@
-# SNS launch steps
+# Commands & actions to go through SNS launch
+
+
+<!---
+CHANGE TITLE: Commands & actions to go through SNS launch - legacy flow 
+NEW: INFO 
+This page refers to a legacy flow of how an SNS is launched. This is still supported at the moment but might be deprecated going forward. It is recommended to use the new SNS launch flow which can be fund [here](./launch-steps-1proposal.md). Please find more context on this topic [here](./index.md).
```
Comment (jessiemongeon1) [inline docs/developer-docs/integrations/sns/launching/launch-steps.md:7]:
```suggestion
[//]: # This page refers to a legacy flow of how an SNS is launched. This is still supported at the moment but might be deprecated going forward. It is recommended to use the new SNS launch flow which can be fund [here](./launch-steps-1proposal.md). Please find more context on this topic [here](./index.md).
```

File: docs/developer-docs/integrations/sns/launching/launch-steps.md [hunk 16]
```diff
@@ -1,9 +1,16 @@
...
+-->
```
Comment (jessiemongeon1) [inline docs/developer-docs/integrations/sns/launching/launch-steps.md:8]:
```suggestion

```

File: docs/developer-docs/integrations/sns/launching/launch-summary.md [hunk 17]
```diff
@@ -1,4 +1,13 @@
 # Stages of an SNS launch
+
+
+<!---
```
Comment (jessiemongeon1) [inline docs/developer-docs/integrations/sns/launching/launch-summary.md:4]:
```suggestion

```

File: docs/developer-docs/integrations/sns/launching/launch-summary.md [hunk 18]
```diff
@@ -1,4 +1,13 @@
...
+CHANGE TITLE: Stages of an SNS launch - legacy flow
```
Comment (jessiemongeon1) [inline docs/developer-docs/integrations/sns/launching/launch-summary.md:5]:
```suggestion
[//]: # CHANGE TITLE: Stages of an SNS launch - legacy flow
```

File: docs/developer-docs/integrations/sns/launching/launch-summary.md [hunk 19]
```diff
@@ -1,4 +1,13 @@
...
+NEW: INFO
```
Comment (jessiemongeon1) [inline docs/developer-docs/integrations/sns/launching/launch-summary.md:6]:
```suggestion
[//]: # NEW: INFO 
```

File: docs/developer-docs/integrations/sns/launching/launch-summary.md [hunk 20]
```diff
@@ -1,4 +1,13 @@
 # Stages of an SNS launch
+
+
+<!---
+CHANGE TITLE: Stages of an SNS launch - legacy flow
+NEW: INFO 
+This page refers to a legacy flow of how an SNS is launched. This is still supported at the moment but might be deprecated going forward. It is recommended to use the new SNS launch flow which can be fund [here](./launch-summary-1proposal.md). Please find more context on this topic [here](./index.md).
```
Comment (jessiemongeon1) [inline docs/developer-docs/integrations/sns/launching/launch-summary.md:7]:
```suggestion
[//]: # This page refers to a legacy flow of how an SNS is launched. This is still supported at the moment but might be deprecated going forward. It is recommended to use the new SNS launch flow which can be fund [here](./launch-summary-1proposal.md). Please find more context on this topic [here](./index.md).
```

File: docs/developer-docs/integrations/sns/launching/launch-summary.md [hunk 21]
```diff
@@ -1,4 +1,13 @@
...
+-->
```
Comment (jessiemongeon1) [inline docs/developer-docs/integrations/sns/launching/launch-summary.md:8]:
```suggestion

```

File: docs/developer-docs/integrations/sns/tokenomics/preparation.md [hunk 22]
```diff
@@ -73,17 +81,17 @@ The developer and airdrop neurons are the only neurons that exist when
...
+Therefore, **if you want to use SNS proposals at this stage, for example to ensure that the dapp can be upgraded 
+at any time, you must make sure that enough of the initial neurons can vote to reach a decison.**
+See [Stage 1 of the launch](../launching/launch-summary.md) for more details of this. 
 :::
 
-All developers, airdrop principals, and the swap participants receive
-their neurons as a *basket of neurons*.
+
+
+<!--- NEW: change link above -->
```
Comment (jessiemongeon1) [inline docs/developer-docs/integrations/sns/tokenomics/preparation.md:92]:
```suggestion
[//]: # NEW: change link above 
```

File: docs/developer-docs/integrations/sns/tokenomics/preparation.md [hunk 23]
```diff
@@ -96,15 +104,22 @@ voting power multiplier increases until it is 1 when all swap tokens have been
 sold.
 
 ### SNS decentralization swap parameters
+<!-- NEW: REMOVE THIS PARAGRAPH -->
```
Comment (jessiemongeon1) [inline docs/developer-docs/integrations/sns/tokenomics/preparation.md:107]:
```suggestion
[//]: # NEW: REMOVE THIS PARAGRAPH 
```

File: docs/developer-docs/integrations/sns/launching/launch-steps.md [hunk 24]
```diff
@@ -1,9 +1,14 @@
-# SNS launch steps
+# Commands & actions to go through SNS launch
+
+
+[//]: # CHANGE TITLE: Commands & actions to go through SNS launch - legacy flow
```
Comment (jessiemongeon1) [inline docs/developer-docs/integrations/sns/launching/launch-steps.md:4]:
```suggestion
<!---
CHANGE TITLE: Commands & actions to go through SNS launch - legacy flow 
```

File: docs/developer-docs/integrations/sns/launching/launch-steps.md [hunk 25]
```diff
@@ -1,9 +1,14 @@
-# SNS launch steps
+# Commands & actions to go through SNS launch
+
+
+[//]: # CHANGE TITLE: Commands & actions to go through SNS launch - legacy flow 
+[//]: # NEW: INFO
```
Comment (jessiemongeon1) [inline docs/developer-docs/integrations/sns/launching/launch-steps.md:5]:
```suggestion
NEW: INFO 
```

File: docs/developer-docs/integrations/sns/launching/launch-steps.md [hunk 26]
```diff
@@ -1,9 +1,14 @@
-# SNS launch steps
+# Commands & actions to go through SNS launch
+
+
+[//]: # CHANGE TITLE: Commands & actions to go through SNS launch - legacy flow 
+[//]: # NEW: INFO 
+[//]: # This page refers to a legacy flow of how an SNS is launched. This is still supported at the moment but might be deprecated going forward. It is recommended to use the new SNS launch flow which can be fund [here](./launch-steps-1proposal.md). Please find more context on this topic [here](./index.md).
```
Comment (jessiemongeon1) [inline docs/developer-docs/integrations/sns/launching/launch-steps.md:6]:
```suggestion
This page refers to a legacy flow of how an SNS is launched. This is still supported at the moment but might be deprecated going forward. It is recommended to use the new SNS launch flow which can be fund [here](./launch-steps-1proposal.md). Please find more context on this topic [here](./index.md).

-->
```

File: docs/developer-docs/integrations/sns/launching/launch-summary.md [hunk 27]
```diff
@@ -1,4 +1,11 @@
 # Stages of an SNS launch
+
+
+[//]: # CHANGE TITLE: Stages of an SNS launch - legacy flow
```
Comment (jessiemongeon1) [inline docs/developer-docs/integrations/sns/launching/launch-summary.md:4]:
```suggestion
<!---
CHANGE TITLE: Stages of an SNS launch - legacy flow
```

File: docs/developer-docs/integrations/sns/launching/launch-summary.md [hunk 28]
```diff
@@ -1,4 +1,11 @@
...
+[//]: # NEW: INFO
```
Comment (jessiemongeon1) [inline docs/developer-docs/integrations/sns/launching/launch-summary.md:5]:
```suggestion
NEW: INFO 
```

File: docs/developer-docs/integrations/sns/launching/launch-summary.md [hunk 29]
```diff
@@ -1,4 +1,11 @@
 # Stages of an SNS launch
+
+
+[//]: # CHANGE TITLE: Stages of an SNS launch - legacy flow
+[//]: # NEW: INFO 
+[//]: # This page refers to a legacy flow of how an SNS is launched. This is still supported at the moment but might be deprecated going forward. It is recommended to use the new SNS launch flow which can be fund [here](./launch-summary-1proposal.md). Please find more context on this topic [here](./index.md).
```
Comment (jessiemongeon1) [inline docs/developer-docs/integrations/sns/launching/launch-summary.md:6]:
```suggestion
This page refers to a legacy flow of how an SNS is launched. This is still supported at the moment but might be deprecated going forward. It is recommended to use the new SNS launch flow which can be fund [here](./launch-summary-1proposal.md). Please find more context on this topic [here](./index.md).

-->
```

File: docs/developer-docs/integrations/sns/tokenomics/preparation.md [hunk 30]
```diff
@@ -73,17 +81,17 @@ The developer and airdrop neurons are the only neurons that exist when
...
+Therefore, **if you want to use SNS proposals at this stage, for example to ensure that the dapp can be upgraded 
+at any time, you must make sure that enough of the initial neurons can vote to reach a decison.**
+See [Stage 1 of the launch](../launching/launch-summary.md) for more details of this. 
 :::
 
-All developers, airdrop principals, and the swap participants receive
-their neurons as a *basket of neurons*.
+
+
+[//]: # NEW: change link above
```
Comment (jessiemongeon1) [inline docs/developer-docs/integrations/sns/tokenomics/preparation.md:92]:
```suggestion
<!---
NEW: change link above 
-->
```

File: docs/developer-docs/integrations/sns/tokenomics/preparation.md [hunk 31]
```diff
@@ -96,15 +104,22 @@ voting power multiplier increases until it is 1 when all swap tokens have been
 sold.
 
 ### SNS decentralization swap parameters
+[//]: # NEW: REMOVE THIS PARAGRAPH
```
Comment (jessiemongeon1) [inline docs/developer-docs/integrations/sns/tokenomics/preparation.md:107]:
```suggestion
<!---

NEW: REMOVE THIS PARAGRAPH 

-->
```

**Review by sesi200 — APPROVED** [review]

**Pull request merged**